lib/cocoapods-core/specification/linter.rb

```diff
@@ -99,6 +100,20 @@ def validate_root_name
         end
       end
 
+      # Generates a warning if the requires_arc attribute has true or false string values.
+      #
+      # @return [void]
+      #
+      def check_requires_arc_attribute
+        attribute = DSL.attributes.values.find{|attr| attr.name == :requires_arc}
```

Comment:
is the formatting correct here? I think spaces are needed, `rubocop -a` I believe will take care of it for you.